### PR TITLE
OHRM 259 : Prerequisite verification for addons and showing the warning message if prerequisites not met

### PR DESCRIPTION
**Pull request by Bashitha**

### CODE REVIEW

**Comment by samanthajayasinghe** [comment]
@Bashitha need rebasing for this PR

File: symfony/plugins/orangehrmMarketPlacePlugin/test/model/service/MarketplaceServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                        ]
+                    ]
+            ];
+
+
+        $result = $this->marketplaceService->addonPrerequisitesVerify($addon);
+
+        if (extension_loaded("toggl") && extension_loaded("soap") && extension_loaded("image")) {
+            $this->assertEquals(count($result), 0);
+        } elseif (!extension_loaded("toggl") && extension_loaded("soap") && extension_loaded("image")) {
```
Comment (Damith88) [inline symfony/plugins/orangehrmMarketPlacePlugin/test/model/service/MarketplaceServiceTest.php]: improve this code

File: symfony/plugins/orangehrmMarketPlacePlugin/web/js/ohrmAddonSuccess.js [hunk 2]
```diff
@@ -33,6 +33,21 @@ $(document).ready(function () {
     });
     $('.installBtn').live('click', function (event) {
         installId = $(this).attr('addid');
+
+        $.ajax({
+            method: "POST",
+            data: {addonID: installId},
+            url: prerequisiteVerificationUrl,
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmMarketPlacePlugin/web/js/ohrmAddonSuccess.js]: What is this prerequisiteVerificationUrl ? is it MP endpoint ? if so you can't call cross domain ajax requests ..  should be handled via php guzzle GET

Comment (Bashitha) [reply symfony/plugins/orangehrmMarketPlacePlugin/web/js/ohrmAddonSuccess.js]: No Samantha, It is an opensource action. Anyway Damith told that this is not a must to convert it into a get request. That's why I kept it as it is.

**Pull request merged**